mod/likes/views/default/likes/count.php

```diff
@@ -5,30 +5,28 @@
  *  @uses $vars['entity']
  */
 
-$num_of_likes = \Elgg\Likes\DataService::instance()->getNumLikes($vars['entity']);
-$guid = $vars['entity']->guid;
+use Elgg\Likes\DataService;
 
-// display the number of likes
-if ($num_of_likes == 1) {
-	$likes_string = elgg_echo('likes:userlikedthis', [$num_of_likes]);
-} else {
-	$likes_string = elgg_echo('likes:userslikedthis', [$num_of_likes]);
-}
+$entity = $vars['entity'];
+/** @var ElggEntity $entity */
 
-$class = 'elgg-lightbox elgg-non-link';
-if (!$num_of_likes) {
-	$class .= ' hidden';
+$num_likes = DataService::instance()->getNumLikes($entity);
+if ($num_likes == 1) {
+	$likes_string = elgg_echo('likes:userlikedthis', [$num_likes]);
+} else {
+	$likes_string = elgg_echo('likes:userslikedthis', [$num_likes]);
 }
 
 $params = [
 	'text' => $likes_string,
 	'title' => elgg_echo('likes:see'),
-	'class' => $class,
+	'class' => 'elgg-lightbox elgg-non-link',
```

Comment:
This needs to be fixed in elgg_view_menu_item(). In my big rewrite project, there is `output/inline_block` view, which fixes all the inconsitencies in menu rendering (in case you want to salvage that bit)